nodl_python/nodl/schemas/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Canonical Ltd.
```

Comment:
If this is suddenly importable, we need to remove it from public API. Easiest way to do that is to make it `_schemas`.

Reply:
fixed in ac62c82